identity-iota/src/client/client.rs

```diff
@@ -1,44 +1,38 @@
 // Copyright 2020-2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-use core::slice::from_ref;
-use identity_core::common::Url;
-use identity_core::convert::ToJson;
-use iota::client::FindTransactionsResponse;
-use iota::client::GetTrytesResponse;
-use iota::client::Transfer;
-use iota::transaction::bundled::Address;
-use iota::transaction::bundled::BundledTransaction;
-use iota::transaction::bundled::BundledTransactionField;
-
-use crate::chain::AuthChain;
-use crate::chain::DiffChain;
-use crate::chain::DocumentChain;
-use crate::client::ClientBuilder;
-use crate::client::Network;
-use crate::client::TxnPrinter;
-use crate::did::Document;
-use crate::did::DocumentDiff;
-use crate::did::DID;
-use crate::error::Error;
-use crate::error::Result;
-use crate::tangle::Message;
-use crate::tangle::MessageId;
-use crate::utils::bundles_from_trytes;
-use crate::utils::create_address_from_trits;
-use crate::utils::encode_trits;
-use crate::utils::txn_hash_trytes;
+use futures::stream::FuturesUnordered;
+use futures::stream::StreamExt;
+use identity_core::{common::Url, convert::ToJson};
```

Comment:
should consistently use the same style of import.

some form of consistency will be enforced through `rustfmt.toml` eventually

Reply:
ahh I see this was the style from my old PR 😄 should move it to the new style